packages/device_info/lib/device_info.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
```

Comment:
Having a full stack solution is better. It still makes me uncomfortable that we are asking users to setup and monitor multiple crash loggers for their app.

Could we go through https://sentry.io/for/android/ and https://sentry.io/for/cocoa/ (i.e. provide a plugin for sentry.io) and encode the Dart exception in a way that also generates a stable fp?

If not, I agree this is much better than nothing.